### PR TITLE
Update OSLC Query spec status and date

### PR DESCRIPTION
**Pull request by DavidJHoney**

### CODE REVIEW

File: specs/query/oslc-query.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
         oslcVersion: "3.0",
 
         // if you wish the publication date to be other than the last modification, set this
-        publishDate: "2020-07-03",
+        publishDate: "2020-10-01",
```
Comment (berezovskyi) [inline specs/query/oslc-query.html:38]: previous publish date and status shall be set to PSD and 2020-07-03

**Comment by berezovskyi** [comment]
prevVersion shall also be set to https://docs.oasis-open-projects.org/oslc-op/query/v3.0/psd01/oslc-query.html

**Review by berezovskyi — APPROVED** [review]
Approved but please don't merge

**Comment by berezovskyi** [comment]
@DavidJHoney the package we provided OASIS had h2 in it. Please check the generated HTML before uploading:

image

**Comment by DavidJHoney** [comment]
The HTML generated b y Respec…Save snapshot made that change.

**Comment by berezovskyi** [comment]
@ndjc need your help, please!

**Comment by berezovskyi** [comment]
Published by OASIS: https://lists.oasis-open-projects.org/g/oslc-op/topic/oslc_query_version_3_0_is_now/77261711?p=,,,50,0,0,0::recentpostdate%2Fsticky,,,50,2,0,77261711

**Pull request merged**